Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -225,7 +225,12 @@ open class MessageContentCell: MessageCollectionViewCell {
                 fallthrough
             }
         default:
-            origin.y = attributes.cellTopLabelSize.height + attributes.messageTopLabelSize.height + attributes.messageContainerPadding.top
+            if attributes.accessoryViewSize.height > attributes.messageContainerSize.height {
```

Comment:
Can you explain this to me a little bit more? I don't know why it's necessary

Reply:
Hey @JulienKode, thanks for explaining this to me 👍 I think the real issue is in the height calculation for the cell? I'm sure if this PR solves the issue the correct way but I totally understand what you're saying is the problem

Reply:
There may actually not be a great way to solve this as I think more about it

Reply:
@SD10 @JulienKode I think the current state is good, it doesnt break the cell layout as all subviews are rendered within the bounds. I think if we want to develop the reliability for rendering options in this case it could be tabled for a future release.

I think we really need to release 1.1, particularly because of the content inset bug people keep facing and the Swift 4.2 update